buildbot/build_tflite.sh

```diff
@@ -109,22 +115,36 @@ cmake -GNinja -S flatbuffers/src/flatbuffers -B flatbuffers/src/flatbuffers-buil
   -DFLATBUFFERS_BUILD_TESTS:BOOL=OFF
 ninja -C flatbuffers/src/flatbuffers-build install
 
+# gemmlowp
+git clone --filter=tree:0 --no-checkout ${GEMMLOWP_REPOSITORY} gemmlowp/src/gemmlowp
+git -C gemmlowp/src/gemmlowp checkout ${GEMMLOWP_TAG}
+cmake -GNinja -S gemmlowp/src/gemmlowp/contrib -B gemmlowp/src/gemmlowp-build \
+  -DCMAKE_INSTALL_PREFIX:PATH=${PWD}/gemmlowp \
+  -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON
+ninja -C gemmlowp/src/gemmlowp-build install
+
+# ml_dtypes
+git clone --filter=tree:0 --no-checkout ${ML_DTYPES_REPOSITORY} ml_dtypes/src/ml_dtypes
+git -C ml_dtypes/src/ml_dtypes checkout ${ML_DTYPES_TAG}
+
 # tflite
 git clone --filter=tree:0 --no-checkout ${TENSORFLOW_REPOSITORY} tensorflow/src/tensorflow
 git -C tensorflow/src/tensorflow checkout ${TENSORFLOW_TAG}
 cmake -GNinja -S tensorflow/src/tensorflow/tensorflow/lite -B tensorflow/src/tensorflow-build \
   -DCMAKE_FIND_PACKAGE_PREFER_CONFIG:BOOL=ON \
   -DCMAKE_INSTALL_PREFIX:PATH=${PWD}/tensorflow \
   -DCMAKE_INSTALL_LIBDIR=lib \
-  -DCMAKE_FIND_PACKAGE_PREFER_CONFIG:BOOL=ON \
```

Comment:
What's the motivation behind this change? The intention behind this option is make CMake try "Config" mode before "Module" mode which is desirable for a hermetic build.

Reply:
It was leftover from debugging, didn't mean to include it in the final diff. Sorry about that. The build after this patch is slightly less hermetic as `libpthreadpool` is new system dependency. There were some notes here https://github.com/tensorflow/tensorflow/commit/8023633ce4caf48859c863e005edc1526089a191 saying that it wasn't possible to provide a prebuilt version and I couldn't get the source folder option working, but didn't dig too deeply into it.